### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by lukasmalkmus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,19 +34,23 @@ make # Run code generators, linters, sanitizers and test suits
 
 ## Authentication
 
-The client is initialized with an access token and the users organization ID
-when using Axiom Cloud or with the URL of the deployment and an access token
-when using Axiom Selfhost. The organization ID can be omitted in case an API
-token is used.
+The client is initialized with an access token. The access token can be an API
+token retrieved from the settings page of the Axiom deployment or a personal
+token retrieved from the users profile page.
```
Comment (bahlo) [inline README.md:39]: Kind of confusing as Profile is also in Settings

File: README.md [hunk 2]
```diff
@@ -34,19 +34,23 @@ make # Run code generators, linters, sanitizers and test suits
...
 
-The access token can an API token retrieved from the settings page of the Axiom
-deployment or a personal token retrieved from the users profile page.
+When using Axiom Cloud the users organization ID must be provided in case a
+personal token is used. The organization ID can be omitted in case an API token
+is used.
```
Comment (bahlo) [inline README.md:43]: Where do you get the organization ID from?

File: README.md (hunk 1 above)
Comment (thecraftman) [reply README.md:39]: API token: settings -> API Token
Personal Token: Settings -> profile -> add personal token

**Comment by lukasmalkmus** [comment]
Addressed your comments @thecraftman @bahlo.

**Review by bahlo — APPROVED** [review]

**Pull request merged**